cfgmgr/intfmgr.cpp

```diff
@@ -115,6 +144,16 @@ void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
     int ret = swss::exec(cmd.str(), res);
     if (ret)
     {
+        /*
+            ipv6 setting on the iface might still haven't been enabled by SDK.
+            Retry adding Ipv6 Prefix after enabling the setting
+        */
+        if (retryV6 && enableIpv6Flag(alias))
+        {
+            SWSS_LOG_INFO("Setting disable_ipv6 flag to 0 for iface: %s", alias.c_str());
```

Comment:
The log does not reflect what this code block is doing

Reply:
Moved to the relevant method